.coveragerc

```
@@ -2,15 +2,16 @@
 
 [report]
 # Regexes for lines to exclude from consideration
-exclude_lines =
-    # Have to re-enable the standard pragma:
-    pragma: no cover
-
-    # Don't complain if non-runnable code isn't run:
+exclude_also =
+    # Don't complain if non-runnable code isn't run
     if 0:
     if __name__ == .__main__.:
+    # Don't complain about code that shouldn't run
+    def create_lut():
     # Don't complain about debug code
     if DEBUG:
+    # Don't complain about compatibility code
+    class TypeGuard:
```

What do you think about this?
```suggestion
    # Don't complain about missing optional dependencies
    except ImportError:
```
Most of these blocks are missing coverage in the tests already, and I'm not sure how much value there would be in adding a CI job with no optional dependencies.

---

This should cover both, without the colon:
```suggestion
    # Don't complain about compatibility code for missing optional dependencies
    except ImportError
```

---

Looking at https://app.codecov.io/gh/python-pillow/Pillow/pull/7666/indirect-changes, it seems to have worked.